libp2p/debugutils.nim

```diff
@@ -124,30 +125,30 @@ proc decodeDumpMessage*(data: openArray[byte]): Option[ProtoMessage] =
     elif value == uint64(Incoming):
       Incoming
     else:
-      return none[ProtoMessage]()
+      return Opt.none(ProtoMessage)
 
   let res7 = pb.getField(7, pmsg.message)
-  if res7.isErr() or not(res7.get()):
-    return none[ProtoMessage]()
+  if res7.isErr() or not(res7.get(false)):
+    return Opt.none(ProtoMessage)
 
   value = 0'u64
   let res1 = pb.getField(1, value)
-  if res1.isOk() and res1.get():
-    pmsg.seqID = some(value)
+  if res1.isOk() and res1.get(false):
```

Comment:
is `res1.isOk() `necessary?

Reply:
No, but this file is a bit useless, fixed anyway